erlang_sample/README.md

```
@@ -0,0 +1,47 @@
+Bismut
+======
+This is a hierarchical colored work tracker. Like a traditional bug/task tracker but with multicolored hierarchical items so your whole project looks a bit like a bismuth crystal, and you can see its status immediately just by looking at the tasks at hand. 
+
+![Bismuth crystal](https://upload.wikimedia.org/wikipedia/commons/thumb/c/cc/Bismuth_%285109110719%29.jpg/800px-Bismuth_%285109110719%29.jpg)
+
+*By David Abercrombie, [CC BY-SA 2.0](https://creativecommons.org/licenses/by-sa/2.0), via Wikimedia Commons*
+
+If you get the concept but want to suggest a feature or a bug fix, contact me at: akalenuk@gmail.ua.
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)